kvrocks.conf

```diff
@@ -726,6 +726,13 @@ rocksdb.max_bytes_for_level_base 268435456
 # Default: 10
 rocksdb.max_bytes_for_level_multiplier 10
 
+# If yes, RocksDB will prefetch some of data asynchronously.
+# RocksDB apply it if reads are sequential and its internal automatic
+# prefetching.
```

Comment:
```suggestion
# If yes, RocksDB will try to read asynchronously and in parallel as much as possible to hide IO latency.
# This feature only takes effect in Iterators and MultiGet.
# In iterators, it will prefetch data asynchronously in the background for each file being iterated on. 
# In MultiGet, it will read the necessary data blocks from those files in parallel as much as possible.
```

Reply:
@PragmaTwice Does this comment look better?

Reply:
Sure, it looks more clear to me.